ports/nrf/boards/common.template.ld

```diff
@@ -70,7 +70,6 @@ SECTIONS
         _etext = .;        /* define a global symbol at end of code */
     } >FLASH_FIRMWARE
 
-    /*
```

Comment:
What is extab?

Reply:
extab is the exception table, which is used in unwinding support on ARM. I thought maybe the exception system of circuit python used this table, which is overflowing enough that it is no longer fitting inline. I noticed that atmel-samd stores the extab in a similar location, so I simply emulated it here.

If circuit python isn't using this data at all, it's safe to discard this section. We could also try building with `-fno-unwind-tables` which should also prevent the generation of this section.

Reply:
Huh, interesting. Is it needed for C++? Maybe we should leave it because we should add C++ library support at some point.

Reply:
It's used for printing out stack unwinding. For example, in Rust, if you discard this section it can't print a line number and function name when you `panic!()`.

If Circuit Python wants to start printing stack unwinding information for C or C++ modules, then this data should be saved. Otherwise, it should be safe to discard.